workspaces/global-floating-action-button/plugins/global-floating-action-button/src/utils.test.tsx

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as React from 'react';
+import {
+  evaluateFloatingButtonsWithPositions,
+  filterAndSortButtons,
+  sortButtonsWithPriority,
+} from './utils';
+
+const floatingButtons = [
+  {
+    color: 'success',
+    icon: <>Some Icon</>,
```

Comment:
If you allow the string, you can rename this file to utils.test.ts and remove the React dependency. :)